guac.yaml

```diff
@@ -33,3 +33,5 @@ gql-addr: http://localhost:8080/query
 # Collector behavior
 service-poll: true
 use-csub: true
+
+log-level: 0
```

Comment:
Default log level probably should be info. Would prefer using names in config to numbers

Reply:
Internal representation can still be an enum

Reply:
Made this change. The env var and the config var can now be set to `debug`, `info`, `warn`, `error`, `dpanic`, `panic`, or `fatal`.